app/components/blacklight/gallery/openseadragon_embed_component.rb

```diff
@@ -12,6 +12,7 @@ def initialize(document:, presenter:, view_config: nil, classes: [], **kwargs)
         @presenter = presenter
         @view_config = view_config
         @classes = classes
+        @id_prefix ||= id_prefix
```

Comment:
This doesn't need to be `||=` because it can never be assigned prior to the initializer running.